CHANGELOG.md

```diff
@@ -1,9 +1,7 @@
 Changelog
 =========
 
-dev-master
-----------
-
+* **2016-06-18**: [BC BREAK] Removed all `*.class` parameters.
```

Comment:
Unrelated, but I think we should change this convention for 2.0 and use the short git hash instead -- the date is irrelevant, the git hash has much more value, of course it might change when we squash causing more hassle. Maybe remove the reference altogther and just date the releases.